sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -47,15 +45,19 @@
 from devtools_testutils import ResourceGroupPreparer, StorageAccountPreparer
 from _shared.testcase import StorageTestCase, GlobalStorageAccountPreparer, GlobalResourceGroupPreparer
 
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 TEST_CONTAINER_PREFIX = 'container'
 TEST_BLOB_PREFIX = 'blob'
-#------------------------------------------------------------------------------
 
+# ------------------------------------------------------------------------------
 class StorageCommonBlobTest(StorageTestCase):
-
-    def _setup(self, storage_account, key):
-        self.bsc = BlobServiceClient(self.account_url(storage_account, "blob"), credential=key)
+    def _setup(self, storage_account, key, additional_policies=None, **kwargs):
+        total_retries = kwargs.pop('total_retries', 10)
+        self.bsc = BlobServiceClient(
+            self.account_url(storage_account, "blob"),
+            credential=key,
+            retry_total=total_retries,
+            _additional_pipeline_policies=additional_policies)
```

Comment:
you could try using _additional_pipeline_policy instead of request hook.

Reply:
This extra param is actually no longer needed in `def _setup()` since I am neither using an additional policy (using a fake token directly now) nor am I using request hooks. I will revert this.